samples/ClientSample/Program.cs

```diff
@@ -7,6 +7,8 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
```

Comment:
Yeah, way ahead of you :). I'm bringing the socket sample back as a commented out chunk of code for now. I don't want to overload us with samples...